yt/fields/derived_field.py

```diff
@@ -250,10 +250,7 @@ def get_dependencies(self, *args, **kwargs):
         This returns a list of names of fields that this field depends on.
         """
         e = FieldDetector(*args, **kwargs)
-        if self._function.__name__ == "<lambda>":
```

Comment:
So I've done some thinking, and I *think* this comes from waaaay back in the day when we may have used lambdas to represent on-disk fields.